src/Verilator.cpp

```diff
@@ -563,6 +611,12 @@ int main(int argc, char** argv, char** env) {
         exit(0);
     }
 
+    if (v3Global.opt.noVerilate() && v3Global.opt.build()) {
```

Comment:
Instead of execBuild in two places (here and below), wrap the verilator stuff in if (v3Global.opt.verilate()) and move the guts to a new function.

This way if we add other steps in the future the overall code path is the same.

Reply:
I see.
Done in 38af6bbae435060505b40f6cc94320698c14809d